README.md

```diff
@@ -1 +1,2 @@
-# test_github_tutorial
\ No newline at end of file
+# test_github_tutorial
```

Comment:
Need better casing for better readability

Reply:
ok